webcompat/static/css/src/filter.scss

```diff
@@ -12,57 +12,57 @@
 }
 
 .filter-label .needstriage {
-  border-color: var(--label-needs-triage);
+  border-color: $label-needs-triage;
 }
 
 .filter-label .needstriage:hover,
 .filter-label .needstriage.is-active {
-  background-color: var(--label-needs-triage);
+  background-color: $label-needs-triage;
 }
 
 .filter-label .needsdiagnosis {
-  border-color: var(--label-needs-diagnosis);
+  border-color: $label-needs-diagnosis;
 }
 
 .filter-label .needsdiagnosis:hover,
 .filter-label .needsdiagnosis.is-active {
-  background-color: var(--label-needs-diagnosis);
+  background-color: $label-needs-diagnosis;
 }
 
 .filter-label .needscontact {
-  border-color: var(--label-needs-contact);
+  border-color: $label-needs-contact;
 }
 
 .filter-label .needscontact:hover,
 .filter-label .needscontact.is-active {
-  background-color: var(--label-needs-contact);
+  background-color: $label-needs-contact;
 }
 
 .filter-label .ready {
-  border-color: var(--label-outreach);
+  border-color: $label-outreach;
 }
 
 .filter-label .ready:hover,
 .filter-label .ready.is-active {
-  background-color: var(--label-outreach);
+  background-color: $label-outreach;
 }
 
 .filter-label .sitewait {
-  border-color: var(--label-site-contacted);
+  border-color: $label-site-contacted;
 }
 
 .filter-label .sitewait:hover,
 .filter-label .sitewait.is-active {
-  background-color: var(--label-site-contacted);
+  background-color: $label-site-contacted;
 }
 
 .filter-label .close {
-  border-color: var(--label-closed);
+  border-color: $label-closed;
 }
 
 .filter-label .close:hover,
 .filter-label .close.is-active {
-  background-color: var(--label-closed);
+  background-color: $label-closed;
 }
 
 @media screen and (min-width: 960px) {
```

Comment:
is there a reason why we did not add a var here?

Reply:
> However there were some media queries not using the default breakpoints. I haven't changed them.

See in the PR description 😄

Reply:
cool, thanks!
Was just wondering as we are using calculations on some of the other breaking points, so this would also make sense right? :)

Reply:
We should definitely check and challenge the existing media queries.

Reply:
Opened #2417 for this.